src/meta/model_v2/migration/src/lib.rs

```diff
@@ -5,6 +5,7 @@ pub use sea_orm_migration::prelude::*;
 mod m20230908_072257_init;
 mod m20231008_020431_hummock;
 mod m20240304_074901_subscription;
+mod m20240322_214331_session_params;
```

Comment:
Do we have to make it a new file?

Reply:
Yes. This is how migration works

Reply:
It seems not: https://github.com/risingwavelabs/risingwave/pull/16090#discussion_r1549000598

Reply:
👀  The PR will not be included in `1.8`, so we need to make it a new file. For example, if this feature is in `1.8.1`, when upgrading from `1.8` to `1.8.1`, `m20240322_214331_session_params` will be automatically applied. If put it inside a previous one, it will be ignored.

Reply:
> 👀 The PR will not be included in `1.8`, so we need to make it a new file.

Didn't notice that. Makes sense to me.